src/coldfront_plugin_api/scim_v2/groups.py

```diff
@@ -117,3 +126,24 @@ def _set_user_status_on_allocation(allocation, user, status):
                 status=AllocationUserStatusChoice.objects.get(name=status)
             )
         return au
+    
+    @staticmethod
+    def _set_user_status_on_project(project, user, status, role, enable_notifications):
+        pu = ProjectUser.objects.filter(
+            project=project,
+            user=user
+        ).first()
+
+        if pu:
+            pu.status = ProjectUserStatusChoice.objects.get(name=status)
```

Comment:
is it necessary to `pu.save()` here? I see that in the similar function above I'm calling that method for `au`.

Reply:
Apparently I don't need to save for `pu`. I don't know why that's the case. Reading the docs doesn't indicate that the save() function saves every object created, not just the object that called it, but I guess we don't have time to look into that?

Reply:
https://docs.djangoproject.com/en/5.0/topics/db/queries/#saving-changes-to-objects indicates that objects aren't saved until `save()` is called. So in this case I would advise erring on the side of caution and calling `save()`.